docs/getting_started.md

```diff
@@ -0,0 +1,67 @@
+# Getting Started with ComposeDB
+
+## Create a schema
+
+* Write a graphQL compatible composite schema in a text file, such as
+
+example: task.schema
+```
+type Task @model(
+    accountRelation: LIST,
+    description: "Basic task"
+) {
+    assignee: String! @length(min:1, max: 30)
+    completed: Boolean
+    content: String! @length(max:2000)
+}
+
+```
+
+* generate a DID key 
+
+If you do not already have a DID key handy, you will need to create one 
+
+If you have downloaded and installed this repo or the [js-glaze](https://github.com/ceramicstudio/js-glaze) repo you can run
+```
+cd packages/cli
+node bin/run.js did:generate-seed
+```
+save this result for use as the did key in the next step.
+
+* Create schema and output encoded composite
+
+`composedb composite:create [graphql_def_file] -k [didkey] -o [output_file.json]`
```

Comment:
USAGE
  $ composedb composite:create INPUT

ARGUMENTS
  INPUT  a path to file containing valid ceramic composite definition in GraphQL Schema Definition Language

OPTIONS
  -c, --ceramic-url=ceramic  Ceramic API URL
  -k, --did-private-key=key  DID Private Key
  -o, --output               a path to file where the resulting encoded composite definition should be saved
```